server/events/plan_command_runner.go

```diff
@@ -119,6 +119,12 @@ func (p *PlanCommandRunner) runAutoplan(ctx *command.Context) {
 		ctx.Log.Warn("unable to update plan commit status: %s", err)
 	}
 
+	if baseRepo.VCSHost.Type == models.Gitlab {
+		if err := p.commitStatusUpdater.UpdateCombinedCount(ctx.Log, ctx.Pull.BaseRepo, ctx.Pull, models.PendingCommitStatus, command.Apply, 0, len(projectCmds)); err != nil {
+			ctx.Log.Warn("unable to update apply commit status: %s", err)
+		}
+	}
```

Comment:
can we also add test for this?

Reply:
I am not a developer and not familiar with writing tests

Reply:
okay, I can take a look at this later. but you can help with the deployment testing, right?

Reply:
Already build and launched on 2 environments. works fine with gitlab and github

Reply:
Oh nice!